README.md

```diff
@@ -321,12 +323,26 @@ In order to build llama.cpp you have three different options.
 - Using `CMake`:
 
     ```bash
-    mkdir build
-    cd build
-    cmake ..
-    cmake --build . --config Release
+    cmake -B build
+    cmake --build build --config Release
     ```
 
+    **Note**: for `Debug` builds, there are two cases:
+
+    - General case (esp. for default Makefile or Ninja generation):
+
+      ```bash
+      cmake -B build -DCMAKE_BUILD_TYPE=Debug
+      cmake --build build
+      ```
+
+    - Special case for multi-config generators (`-G` param set to Visual Studio, XCode...; note that `--config` is ignored by other generators):
```

Comment:
Is there any reference that considers single-config generators as the general case, and multi-config generators as a special case? Otherwise, I would prefer more neutral language (ie. one case for single-config generators, and another case for multi-config generators, without additional qualifiers).

Reply:
Good point, reworded this, thanks!